ConnectingToDB/Program.cs

```diff
@@ -5,18 +5,18 @@ namespace ConnectingToDB
 {
     class Program
     {
-        static void Main()
+        static void ShowAll()
         {
             var connection = "Server=mysql60.hostland.ru;Database=host1323541_shambala1;Uid=host1323541_itstep;Pwd=269f43dc;";
             var db = new MySqlConnection(connection);
             db.Open();
 
             var sql = "SELECT login, name, email FROM table_account JOIN table_user ON table_account.id = table_user.id WHERE table_account.is_active = FALSE;";
```

Comment:
```suggestion
            var sql = "SELECT login, name, email FROM table_account JOIN table_user ON table_account.id = table_user.id;";
```
Изменить запрос, удаление условий выборки данных БД